### ISSUE TITLE
1.17 release team meeting calendar event has broken hyperlink

### ISSUE THREAD

**Issue by tpepper** [kubernetes/sig-release#840]
It looks like the meeting invite "[1.17] Kubernetes Weekly Release Team Meeting" has the correct zoom url in text:

https://zoom.us/j/406805785

But the hyperlink behind that text points to a different, incorrect zoom meeting.  The invite needs updated with an edit to the link to insure the text and the link both go to:

https://zoom.us/j/406805785

**Comment by tpepper** [comment]
I went to this link:
https://github.com/kubernetes/sig-release/tree/master/releases/release-1.17

There I clicked 'Kubernetes Release Calendar' which goes to:
https://calendar.google.com/calendar/embed?src=agst.us_b07popf7t4avmt4km7eq5tk5ao@group.calendar.google.com

There I see the weekly Monday meeting.  If I click on it a dialog box comes up showing:
"Description: https://zoom.us/j/406805785"

But if I mouse over on that zoom link, my browser shows it is hyperlinked to meeting id 327142148.

And clicking on the link indeed zoom takes me to https://zoom.us/j/327142148.

I'm not sure what you updated, but it wasn't the official calendar invite as the above still reproduces, and I did not receive a meeting update from you.
